### ISSUE TITLE
Explorer department/acess rework

### ISSUE THREAD

**Issue by TheDavestDave** [VOREStation/VOREStation#4604]
#### Brief description of the issue
I think it's still unfair that explorers and pathfinders are the only people in the science department but are the only people with out R and D access ((And firing range access)). the frontier phaser, even with the planed nerfs, are much more powerful than the AEG. Everything else should just be helping the department like the SAR does with medical.  At this point I think it's just because of a hate boner for explorers. Roboticists, and xenobiologist....even scientist (but it's their job) regularly print contraband and take it outside science. Yet they have R and D access despite not being part of their jobs. I don't mean knifes and stuff. I mean driveing armed mechs into bar. Shooting the HoS and bitching on coms because it "was just a prank bro". Yet non-scientists still have access to R and D.

Most of exploration is heavily disconnected from the rest of the department because we can't be useful, and it's a common job for going on and lewding. So a lone explorer can't do anything (and fuck off with you can't be expected to do you job regularly trope)  because no access to anything else, and the rest of the department off fucking, or just having a hate boner. Even some RDs just spend the time screaming at explorers for getting basic things like welding goggles, or them being in the circuit room (because we have access to that but not the shooting range.) As well as making shit up to stop us going. Like 3 people isn't enough, oh I meant four....erm you have four? five....can't go because I say so. So I have to go get an IAA involved instead of doing my job.  The rare time we bring a high tech level item back. Everyone is already off fucking, so the only way to get it worked on is to break in ourselves.

The other problem is that no one does paper work in the department, the only thing that connects explorers to science. So gameplay wise it has more to do with sec because of how combat focused it is. To the point where most explorer characters ARE sec. But it's far more common to complain about explorers not doing paperwork.

I think we're best moving exploration in its own department if admins are hell bent not letting explorers act like normal science staff. Gameplay wise they are litteraly sec with smarts. Gameplay wise they feel like sec that have to write reports after they are done.  Although that is due to combat focus maps, with little passive wildlife. No access to tools to dig up fossils and stuff. No Inclinometer, quadrates, sample bags, or other survey gear.

copy paste for discord
"NanakiToday at 4:17 PM
all science roles have a wait period of 7 to 14 days
except for Explorer"

#### What you expected to happen
Equal enforcement for laws and s rules despite of job title.
Admin involvement with players who don't RP

#### What actually happened
Punish everyone instead of job banning offenders.

#### Steps to reproduce
- (Step 1) make a new character eager to explore the unknown and see new sights
- (Step 2) get shat on by everyone for things others did

#### Code Revision
- (Found using the "Show Server Revision" verb under the OOC tab.)
 1fe04192c11e2ad7357512d5a873c44a4ea687d3
#### Anything else you may wish to add:
- (Location if it's a mapping issue, screenshots, sprites, etc.)
tl;dr

there's a lot of shitter in science

people only care if explorers are shitters

a whole job title gets punished for the shitters instead of shitters getting job bans or other admin intervention

due to a lack of science RP the paperwork, the only thing that connects them isn't there. So we're better off moving them to  a new department.

I literally can't do any survey work. I don't even have two poles of known length and a measuring tape.

**Comment by Very-Soft** [comment]
The Explorers, Pathfinders only actual function is relating to the shuttle, and potentially gateway things if they're active.

Exploration doesn't really need access to R&D or really any of the Research wing to do their jobs as they have the weapons and equipment they need to do their jobs provided to them at round start.
 
The SaR gets basic medical access because they are in essence just a paramedic that's assigned to the exploration team. They need access to medical to gather supplies to keep the exploration team alive, and in general can function as paramedics in general when not out on a mission.

Now, while I can understand that some people treat the exploration team harshly, in my experience watching the goings on of the station, there's usually some justification for people not wanting to help, usually coming from the people on the exploration team being rude, overly demanding, and impatient. Where typically when the exploration team has been polite and patient, the people around them are pretty willing to help.

The explorers don't have access to R&D because it was repeatedly abused in the past the the point that people started obsessing over it, and taking over the regular science jobs, while also being the cool guys who get to go on the shuttle.

The pilots lost their cargo access VERY QUICKLY because of the exploration team's constant abuse of that access to make unreasonable orders that would use up all of cargo's resources for hours at a time.

Where on the other hand, roboticists basically need someone to do R&D in order to do a fair portion of the things they are capable of doing. And while I don't really know why say, a xenobiologist/xenobotanist might want anything from R&D, they at least actually work within the research wing.

Explorers/Pathfinders don't. They go on away missions, That's their only actual function, and they start with the tools to do that.

**In short**, the exploration team is meant to be a collaboration between departments. It's evolved from that into mostly something that can stand on its own, but can be enhanced by other departments. Being that that is now the case, I actually have no idea why the explorers even need access to science at all considering their present starting loadout, 

I think if you gave them access to R&D, it would result in more explorers using their equipment to run into mining to do the miners' work, so that they can get materials for R&D, so that they can get better guns so that they can roll over away missions even easier than they already do.

If you want the explorers to have access to R&D's goodies, then it'd be a good idea to make pals with science folk and miners, rather than trying to do everything yourself.

**Comment by Very-Soft** [comment]
I should also note, that if a head is telling you that you can't do your job, and they don't have a good reason, you're 1000% encouraged to ahelp it. It's rediculous for people to just prevent you from doing your job when you've got a team together. IIRC it only actually requires 2 people to take the shuttle out. And if you've got 2 explorers and a SAR, imo there's no reason they should try to stop you unless you're waiting for something specific, or there's some other good reason, like an emergency situation on the station.
